.github/workflows/build-rust-injector.yml

```diff
@@ -27,8 +27,9 @@ jobs:
 
       - name: "Build Rust Binary for x86_64 and arm64"
         run: |
-          make build-injector-linux
-          cd src/injector/target
+          cd src/injector
+          make cross-injector-linux
```

Comment:
Saw @lucasrod16's comments about cross below. We should probably consciously decide either yes or no cross for the one we publish.

Reply:
His comment is about the Readme, we should not be running docker containers in GHAs to build rust (rust comes on gha by default) but we can make it so it uses cargo instead.